templates/index.html

```diff
@@ -58,8 +58,8 @@
       <p>To fully test your browser for sub-resource integrity support, please open <a href="https://btoews.github.io/sri_test/">this page</a>.</p>
 
       <div class="sri-test" id="sri-badge">
-        <link rel="stylesheet" href="/badge/will-pass.css">
-        <link rel="stylesheet" href="/badge/must-fail.css" integrity="sha256-INVALIDHASH">
+        <link rel="stylesheet" href="/badge/will-pass.css" integrity="type:text/css {{sri "./public/badge/will-pass.css"}}">
```

Comment:
This one we probably want to leave without an integrity attribute just to ensure that the "unsupported" SVG is always displayed.